FAQ.md

```diff
@@ -1,21 +1,22 @@
 
 # Frequently Asked Questions
-- [Why does emoji break alignment in a Table or Panel?](#why-does-emoji-break-alignment-in-a-table-or-panel)
-- [Why does content in square brackets disappear?](#why-does-content-in-square-brackets-disappear)
-- [python -m rich.spinner shows extra lines](#python--m-rich.spinner-shows-extra-lines)
-- [How do I log a renderable?](#how-do-i-log-a-renderable)
-- [Strange colors in console output.](#strange-colors-in-console-output.)
+- [Frequently Asked Questions](#frequently-asked-questions)
+  - [Why do emoji break alignment in a Table or Panel?](#why-do-emoji-break-alignment-in-a-table-or-panel)
+  - [Why does content in square brackets disappear?](#why-does-content-in-square-brackets-disappear)
+  - [python -m rich.spinner shows extra lines](#python--m-richspinner-shows-extra-lines)
+  - [How do I log a renderable?](#how-do-i-log-a-renderable)
+  - [Strange colors in the console output.](#strange-colors-in-the-console-output)
 
 <a name="why-does-emoji-break-alignment-in-a-table-or-panel"></a>
-## Why does emoji break alignment in a Table or Panel?
+## Why do emoji break alignment in a Table or Panel?
 
-Certain emoji take up double space within the terminal. Unfortunately, terminals don't always agree how wide a given character should be.
+Certain emoji take up double space within the terminal. Unfortunately, terminals don't always agree on how wide a given character should be.
 
 Rich has no way of knowing how wide a character will be on any given terminal. This can break alignment in containers like Table and Panel, where Rich needs to know the width of the content.
 
-There are also *multiple codepoints* characters, such as country flags, and emoji modifiers, which produce wildly different results across terminal emulators. 
+There are also *multiple codepoints* characters, such as country flags, and emoji modifiers, which produce wildly different results across terminal emulators.
 
-Fortunately, most characters will work just fine. But you may have to avoid using the emojis that break alignment. You will get good results if you stick to emoji released on or before version 9 of the Unicode database,
+Fortunately, most characters will work just fine. But you may have to avoid using emojis that break alignment. You will get good results if you stick to emoji released on or before version 9 of the Unicode database,
```

Comment:
This sentences uses "emojis", but earlier changes use "emoji" as the plural of "emoji" - for example, "Certain emoji take up ..."